src/settings/JobProfiles/tests/ViewJobProfile.test.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import {
+  fireEvent,
+  waitFor,
+} from '@testing-library/react';
+
+import { noop } from 'lodash';
+
+import { renderWithIntl } from '@folio/stripes-data-transfer-components/test/jest/helpers';
+
+import '../../../../test/jest/__mock__';
+import {
+  renderWithReduxForm,
+  translationsProperties,
+} from '../../../../test/jest/helpers';
+
+import { ViewJobProfile } from '../ViewJobProfile';
+
+const jobProfile = {
+  records: [
+    {
+      parentProfiles: [],
+      childProfiles: [],
+      dataType: 'MARC',
+      name: 'Inventory Single Record - Default Create Instance',
+      description: '',
+      id: 'testId',
+      metadata: {
+        createdByUsername: 'System',
+        updatedByUsername: 'System',
+        createdByUserId: '',
+        createdDate: '2021-03-16T15:00:00.000+00:00',
+        updatedByUserId: '',
+        updatedDate: '2021-03-16T15:00:00.000+00:00',
+      },
+      tags: { tagList: [] },
+      userInfo: {
+        firstName: 'System',
+        lastName: 'System',
+        userName: 'System',
+      },
+    }],
+  hasLoaded: true,
+};
+
+const viewJobProfileProps = profile => ({
+  match: { params: { id: 'test id' } },
+  resources: { jobProfile: profile },
+  history: {
+    block: noop,
+    push: noop,
+    replace: noop,
+  },
+  location: {
+    search: '',
+    pathname: '',
+  },
+  onClose: noop,
+  onDelete: noop,
+});
+
+const renderViewJobProfile = ({
+  history,
+  match,
+  location,
+  onClose,
+  onDelete,
+  resources,
+}) => {
+  const component = () => (
+    <Router>
+      <ViewJobProfile
+        resources={resources}
+        location={location}
+        match={match}
+        history={history}
+        tagsEnabled
+        onClose={onClose}
+        onDelete={onDelete}
```

Comment:
```suggestion
        onClose={noop}
        onDelete={noop}
```
If these props don`t change from test to test, why don't you set **noop** directly in this props